rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy

```diff
@@ -1376,6 +1376,22 @@ class ScheduledExecutionController  extends ControllerBase{
                     render ([success: true] as JSON)
                 }
             }
+        } else if(result.errorCode){
```

Comment:
you don't need this section, the problem is just the call to `renderErrorFormat` is breaking the response